redhat-developer/developers.redhat.com
# Fixing diffing revisions on Drupal

Diffs were broken before. This updates the diff module to a more current version that works correctly.

To test this

1) Make a change to some content
2) View the revisions
3) Select the new revision and an old version then click the compare button

You should now see a difference of the changes
